helm/observability-bundle/values.yaml

```diff
@@ -188,13 +188,67 @@ apps:
     chartName: promtail
     catalog: giantswarm
     enabled: false
-    namespace: kube-system
+    namespace: promtail
     # used by renovate
     # repo: giantswarm/promtail-app
     version: 1.0.1
     # User values can be provided via a ConfigMap or Secret for each individual app
-    userConfig: {}
+    userConfig:
```

Comment:
So, TBH, I was thinking it would be easier if we went with the extra-config on the olly bundle level but maybe that's something we need to discuss monday

Reply:
So to come back to this. I think we should not use userConfig here at all, because it gets overwritten completely by customers.

If they want to do 1 change, they would need to copy the whole userconfig which they won't let's be honest.



My idea is that we link the CM and secrets using extraConfigs instead (this makes a lof of sense, trust me :D) because the CMs will be created by our operator anyway.

What I would add in userConfig is something we are fine getting overriden by customers so basically, default values we do not use :)

Reply:
I moved promtail config and secret in extraConfigs.

Reply:
Nice but then we need them to be created. @giantswarm/team-atlas are you find with this and the names? This is what the logging controller will use.

Reply:
BTW, we need to add the cluster-id because as a prefix as it would not work with capi

Reply:
I think you can achieve that using https://helm.sh/docs/chart_template_guide/yaml_techniques/#yaml-anchors

Reply:
CAPI: 
namespace=>org-giantswarm
name=>{cluster-id}-promtail-app-XXX

Vintage:
namespace=>MC=giantswarm, WC=cluster-id
name=>promtail-app-XXX

To be more align, I suggest for Vintage to add {cluster-id} in prefix for the name

Reply:
Yep you've read my mind :)

Reply:
But then I'm thinking, should we not have the cm and secret named cluster-id-logging-config and cluster-id-logging-secret?

Reply:
Yes that's better !

Do you know a way to detect if the bundle is deployed on either MC or WC ?

Reply:
Why do you need this?

Reply:
For Vintage, namespace is `giantswarm` for MC and {cluster-id} for WC :-(

Reply:
That's probably .Release.namespace right? is there no way to get it?